test/js/node/test/common/index.js

```diff
@@ -400,57 +398,6 @@ if (global.Storage) {
   );
 }
 
-if (global.Bun) {
```

Comment:
this doesn't break a bunch of tests?

Reply:
no because the globals check that uses this is also removed